packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -16,7 +16,7 @@ const selectors = {
 	switchToDraftButton: `${ panel } button.editor-post-switch-to-draft`,
 
 	// Preview
-	previewButton: `${ panel } [aria-label="Preview"]:visible`,
+	previewButton: `${ panel } :text("Preview"):visible, [aria-label="Preview"]:visible`,
```

Comment:
I'll have to look at refactoring this, because this is a bit brittle. It will work for now though, because the refctor is complicated by this selector being used in a few places.